app/HomeFavorite.js

```diff
@@ -20,27 +21,43 @@ export default class HomeFavorite extends Component {
     })
     .then(({data}) => {
       let members = data;
-      this.setState({members: this.state.members.cloneWithRows(members)})
+      this.setState({members: this.state.members.cloneWithRows(members)});
     })
     .catch(err => {
       console.log('There was an error fetching members', err);
-    })
-  };
+    });
+  }
 
   render() {
     // const params = this.props.navigation.state.params;
     // const { navigate } = this.props.navigation;
     return (
-      <View>
+      <View style={styles.container}>
         <Button
           onPress={() => console.log("PRESSED")}
-        title="Add Member To Favorite"
+          title="Add Member To Favorite"
         />
         <ListView
           dataSource={this.state.members}
-          renderRow={(rowData) => <Text style={{textAlign: 'center', fontSize: 20}}>{rowData.first}</Text>}
+          style={styles.listView}
+          renderRow={(rowData) => this.renderMembers(rowData) }
         />
       </View>
     );
   }
+
+  renderMembers(memberData) {
+    // const { navigate } = this.props.navigation;
+    console.log(this.props);
```

Comment:
Let's remove this console.log and do we need that commented out code on line 50?